src/App/app.test.js

```diff
@@ -1,5 +1,25 @@
-import { sum } from './sum';
+import React from 'react';
+import App from './';
+import { mount } from 'enzyme';
+import { Button } from '@patternfly/react-core';
+import renderer from 'react-test-renderer';
 
-test('adds 1 + 2 to equal 3', () => {
-  expect(sum(1, 2)).toBe(3);
+describe('App tests', () => {
+  test('App component matches snapshot', () => {
+    const component = renderer.create(<App />);
+    let tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
```

Comment:
I'm assuming this is the case, but just to be sure :)

We're using `renderer` here instead of `mount/shallow` for example sake?

Reply:
Good call. Swapped out for the patternfly standard shallow here.